packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -527,13 +548,7 @@ export default class TabNetwork extends BaseComponent {
       newVif: !this.state.newVif,
     })
 
-  _getIpAddresses = createSelector(
-    () => this.props.vm.addresses,
-    // VM_guest_metrics.networks seems to always have 3 fields (ip, ipv4 and ipv6) for each interface
-    // http://xenbits.xenproject.org/docs/4.12-testing/misc/xenstore-paths.html#attrvifdevidipv4index-ipv4_address-w
-    // https://github.com/xapi-project/xen-api/blob/d650621ba7b64a82aeb77deca787acb059636eaf/ocaml/xapi/xapi_guest_agent.ml#L76-L79
-    addresses => uniq(values(addresses))
-  )
+  _getAddressKeys = createSelector(() => this.props.vm.addresses, keys)
```

Comment:
You can create a collection of addresses by device. Then it will be easy to display the VIF's dedicated addresses.

Example:
```
[
'0': [ `192.168.100.50`, `192.168.100.40`],
'1': []
]
```